app/views/partials/multi_factor_authentication/_mfa_selection.html.erb

```diff
@@ -2,8 +2,8 @@
       'two_factor_options_form[selection][]',
       option.type,
       false,
-      disabled: option.disabled?,
-      checked: option.disabled?,
+      disabled: option.disabled? && option.type === 'piv_cac' || option.disabled? && option.type === 'backup_code' || option.disabled? && option.type === 'phone',
+      checked: option.disabled? && option.type === 'piv_cac' || option.disabled? && option.type === 'backup_code',
```

Comment:
Rather than hard-coding references to specific MFA types, can we move the logic into the presenter for the individual methods themselves?

In other words, I'd assume most of the subclasses of `SelectionPresenter` should define `disabled?` as returning `false` by default, and we override for the PIV / backup code / phone presenters.